src/main/java/client/Rock.java

```diff
@@ -0,0 +1,150 @@
+package client;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.function.Predicate;
+
+import commands.Commands;
```

Comment:
I like how you package everything into different packages according to their functionalities!